.github/workflows/nightly.yaml

```diff
@@ -23,4 +24,42 @@ jobs:
       run: |
         echo $GIT_TAG
         git tag $GIT_TAG
-        git push origin $GIT_TAG
\ No newline at end of file
+        git push origin $GIT_TAG
+
+  image-deps-updater:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
+
+    - name: Setup Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.16.3'
+
+    - name: Run Update Script
```

Comment:
imagedeps optionally uses an environment variable GITHUB_AUTH_TOKEN as an API token to get around rate limiting concerns.  You already have a github secret, might want to set the environment variable here.